### DISCUSSION TITLE
How to disable [edit this page] link in blog

### DISCUSSION THREAD

**Discussion by cocobase** [facebook/docusaurus#9900] [General] [answered]
I am trying to build my own blog site, when link to open one blog article,  the end of article has a link [edit this page]. How can I disable show this link? 
![截屏2024-02-29 下午11 25 06](https://github.com/facebook/docusaurus/assets/60334709/81d28048-e6a3-47d8-8e1d-908092841ad1)
I am using version 3.1.1
Thanks a lot.

**Comment by slorber** [comment]
Just removing the blog `editUrl` site config option should be enough

**Reply by cocobase** [reply, marked as answer]
according your suggestion, It works. Thank you!
![截屏2024-03-02 上午7 56 51](https://github.com/facebook/docusaurus/assets/60334709/0b0ab38c-0354-4719-803b-47e1e02d41ca)
